### PR TITLE
Refactor RSpec for AnnotateModels (3) - AnnotateModels.get_schema_info (with custom options)

### PR DESCRIPTION
**Pull request by nard-tech**
I refactored test cases of `AnnotateModels` with custom options.

### CODE REVIEW

**Comment by nard-tech** [comment]
I will refactor test cases of `AnnotateModels` without custom options in another PR.

**Comment by nard-tech** [comment]
@reviewers
Please check changes commit by commit.
I recommend you to use option "Hide whitespace changes".

**Comment by drwl** [comment]
> @reviewers
> Please check changes commit by commit.
> I recommend you to use option "Hide whitespace changes".

Taking a look now. Good tip on `Hide whitespace changes` 👍 a lot easier to see

File: spec/lib/annotate/annotate_models_spec.rb [hunk 1]
```diff
@@ -954,283 +954,429 @@ def mock_column(name, type, options = {})
...
+                #
+              EOS
+            end
+
+            it 'works with option "hide_default_column_types"' do
+              is_expected.to eq expected_result
+            end
+          end
+
+          context 'when "hide_default_column_types" is "skip"' do
```
Comment (drwl) [inline spec/lib/annotate/annotate_models_spec.rb]: Reviewing this is interesting. I think there's a drift between test code and what we display in options from `lib/annotate/parser.rb`

**Pull request merged**